tests/integ/test_training_compiler.py

```diff
@@ -105,10 +119,10 @@ def test_huggingface_pytorch(
                 "per_device_train_batch_size": 128,
                 "output_dir": "/opt/ml/model",
             },
-            environment={"GPU_NUM_DEVICES": "1"},
```

Comment:
who is going to set this now?

Reply:
Training Toolkit will set this for Distributed Training.
Pytorch_Xla init will default this to 1 for all other cases.